app/services/code_climate/projects_without_c_c.rb

```diff
@@ -0,0 +1,43 @@
+module CodeClimate
+  class ProjectsWithoutCC < BaseService
+    attr_reader :department, :from, :languages
+
+    def initialize(department:, from:, languages:)
+      @department = department
+      @from = from.to_i
+      @languages = languages
+    end
+
+    def call
+      projects_in_given_languages(projects_in_department)
```

Comment:
Are you returning no cc projects within the time range?

Reply:
Based on the current implementation at `CodeClimate::ProjectsDetailsService`, it filters projects by the metrics' `snapshot_time` range, so the same happens here. So, when no-cc projects are returned, it's because their metrics are not within the `snapshot_time: from.weeks.ago..Time.zone.now` range.